src/js/reducers/conversation-reducer.js

```diff
@@ -25,16 +25,9 @@ const sortMessages = (messages) => messages.sort((messageA, messageB) => {
 });
 
 const addMessage = (messages, message) => {
-    const existingMessage = messages.find((m) => isEqual(m, message));
```

Comment:
yeah we figured it was useless. they would be useful when calling `setConversation` but it's only used when the conversation hasn't started yet and has no existing messages.